tasks/src/main/java/ru/sberbank/school/task02/calculator/CompareQuotesBenificiary.java

```diff
@@ -0,0 +1,33 @@
+package ru.sberbank.school.task02.calculator;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+
+import java.util.Comparator;
+
+public class CompareQuotesBenificiary implements Comparator<QuotePrice> {
+    private Beneficiary beneficiary;
+    private ClientOperation operation;
+
+    public CompareQuotesBenificiary (ClientOperation operation, Beneficiary beneficiary) {
+        this.beneficiary = beneficiary;
+        this.operation = operation;
+    }
+
+    @Override
+    public int compare(QuotePrice quote1, QuotePrice quote2) {
+        //Ищем большую цену продажи банком//Offer
+        if ((beneficiary == Beneficiary.BANK && operation == ClientOperation.BUY)
+        || (beneficiary == Beneficiary.CLIENT && operation == ClientOperation.SELL)){
```

Comment:
&#39;if&#39; child has incorrect indentation level 8, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a949837191>*